### PR TITLE
Create a static site from OpenApi spec

### PR DESCRIPTION
**Pull request by guergana**
This PR uses [swagger-ui](https://github.com/swagger-api/swagger-ui/) to generate a static website from the OpenApi Spec.

Available at `http://localhost:<YOUR_PORT>/api-docs`.

I also moved the `mismatchfinder-api-spec.yml` file to avoid duplication. 

Since the package is automatically available from unpkg and the index file is located in the public folder, it should be automatically deployed. We wont need a Github action for deployment. 

Swagger OpenApi Specification can be seen following [this link](https://mismatch-finder-staging.toolforge.org/api-docs/index.html)

Bug: [T287948](https://phabricator.wikimedia.org/T287948)

### CODE REVIEW

File: public/api-docs/index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script src="https://unpkg.com/swagger-ui-dist@3/swagger-ui-bundle.js" charset="UTF-8"></script>
+    <script src="//unpkg.com/swagger-ui-dist@3/swagger-ui-standalone-preset.js"></script>
```
Comment (itamargiv) [inline public/api-docs/index.html:8]: I think we don't need this script anymore

File: public/api-docs/index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <link rel="stylesheet" href="//unpkg.com/swagger-ui-dist@3/swagger-ui.css" />
+    <title>Swagger Mismatch Finder Open Api Docs</title>
+</head>
+<body>
+    <div id="swagger-ui"></div>
```
Comment (itamargiv) [inline public/api-docs/index.html:13]: Nice! no need to create webpack workarounds :D

**Review by itamargiv — CHANGES_REQUESTED** [review]
Fantastic job! Only one tiny change and we're good to go :dancing_men:

File: public/api-docs/index.html (hunk 1 above)
Comment (guergana) [reply public/api-docs/index.html]: True!

**Review by itamargiv — APPROVED** [review]
Great work, thank you!

**Comment by guergana** [comment]
This PR uses swagger-ui to generate a static website from the OpenApi Spec.

Available at http://<url>/api-docs/index.html

Bug: T287948

**Pull request merged**